composables/masto/publish.ts

```diff
@@ -120,6 +120,62 @@ export function useUploadMediaAttachment(draftRef: Ref<Draft>) {
   let failedAttachments = $ref<MediaAttachmentUploadError[]>([])
   const dropZoneRef = ref<HTMLDivElement>()
 
+  const maxPixels
+    = currentInstance.value!.configuration?.mediaAttachments?.imageMatrixLimit
+      ?? 16777216 // 4096x4096
```

Comment:
I think we can do the math, no need to precompute it.
```suggestion
      ?? 4096**2
```